src/UserStory/Entries.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import validate from 'validate.js'
+
+const initialState = {
+    storyFields: { date:"", owner:"", usId:"", title:"", question:"", note:"" },
+    fieldErrors: { dateError:"", ownerError:"", usIdError:"", titleError:"" }
+}
+
+const constraints = {
+    date: {
+        presence: {
+            allowEmpty : false
+        }
+    },
+    owner: {
+        presence: {
+            allowEmpty : false
+        }
+    },
+    usId: {
+        presence: {
+            allowEmpty : false
+        }
+    },
+    title: {
+        presence: {
+            allowEmpty : false
+        }
+    }
+}
+
+class Entries extends Component {
+    constructor(props){
+        super(props)
+        this.state = initialState;
+    }
+
+    validate() {
+        const error = validate(this.state.storyFields, constraints)
+
+        if(error){ 
+            this.setState({
+                dateError: error.date,
+                ownerError: error.owner,
+                usIdError: error.usId,
+                titleError: error.title
+            })
+            return false
+        }
+        return true
+    }
+
+    handleSubmit = (event) => {
+        event.preventDefault()
+        const isValid = this.validate()
+        if (isValid) {
+            console.log("Data taken, reset field")
```

Comment:
In the production application, the `console.log` is not allowed. So please remember to remove them when you finish your testing and before make PRs